exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/DistributedQueryQueue.java

```diff
@@ -273,9 +273,8 @@ public QueueLease enqueue(QueryId queryId, double cost) throws QueryQueueExcepti
     }
 
     if (lease == null) {
-      int timeoutSecs = (int) Math.round(configSet.queueTimeout/1000.0);
-      logger.warn("Queue timeout: {} after {} seconds.", queueName, timeoutSecs);
-      throw new QueueTimeoutException(queryId, queueName, timeoutSecs);
+      logger.warn("Queue timeout: {} after {} ms.", queueName, configSet.queueTimeout);
```

Comment:
This was actually not wrong. Instead, it mapped the number from the internal ms units into more user-friendly units of seconds. That is, it is more useful to see:

Queue timeout: small after 60 seconds

Than:

Queue timeout: small after 60000 ms.

If we do want to use ms. perhaps format them with commas:

Queue timeout: small after 60,000 ms.

Or show both:

Queue timeout: small after 60,000 ms. (60 seconds)

Reply:
I made the change to be consistent with QueryTimeoutException which logs the time in ms.
I made the changes as suggested in both the places.

Queue timeout: small after 60,000 ms. (60 seconds)